### ISSUE TITLE
Shortcuts to the homescreen disappear after reboot

### ISSUE THREAD

**Issue by Zanzibari** [Divested-Mobile/DivestOS-Build#304]
Hi,

I installed D-OS on griffin. Works fine, except, that all shortcuts to the homescreen — of additionally F-Droid installed, non-ROM default apps — are disappearing after each reboot!

**Comment by SkewedZeppelin** [comment]
Are you using an SD card with adoptable storage?

**Comment by Zanzibari** [comment]
Wow, that was fast!
> Are you using an SD card with adoptable storage?
SD card, yes, as additional storage, not portable.
> A different app launcher?
no
> Using one of those app freezers?
no idea, what that is; thus negative
> Any changes made over ADB?
I installed Magisk, coz I want to control/limit my battery charge via Acca. 

Fun fact: even Mulch's shortcut disappears, while Mull's remains…

**Comment by SkewedZeppelin** [comment]
>  SD card, yes, as additional storage, not portable.

This is broken.
Has been for years in AOSP.
I'll add a note to the website.

**Comment by Zanzibari** [comment]
> This is broken.

That far I was using LOSmicroG, and there additional storage was working. Even with encryption. Unfortunately no updates anymore :-(

**Comment by Zanzibari** [comment]
I performed a factory reset. SD card, as additional storage, still present. But lo and behold: shortcuts persist after reboot! I installed an "alien" app: Privacy Browser. Shortcuts remain.

Thereafter I installed AccA — and AccA requires Magisk (which was dormant that far)! 
Result: Magisk and DivestOS don't match! 
That's sad, since charging just to 75% does almost zero wear to my battery.  

At least the problem is identified now — will there be a solution?
